.travis.yml

```diff
@@ -25,11 +25,11 @@ install:
   - bash miniconda.sh -b -p $HOME/miniconda
   - export PATH="$HOME/miniconda/bin:$PATH"
   - conda config --set always_yes yes --set changeps1 no
-  - conda install conda=4.6
+  - conda update conda
   - conda info
 
   # Install dependencies
-  - conda env create --file="${ENV_FILE}"
+  - CONDA_RESTORE_FREE_CHANNEL=1 conda env create --file="${ENV_FILE}"
```

Comment:
```suggestion

  - set CONDA_RESTORE_FREE_CHANNEL=1 
  - conda env create --file="${ENV_FILE}"
```

Reply:
Travis is green, so I think this doesn't matter much (but the conda config is nicer, so will apply this)

Reply:
oh, derp.  I'll take another look at appveyor.